### DISCUSSION TITLE
Provider Destroyer - When my application loads or when I navigate to another section, the provider is destroyed.

### DISCUSSION THREAD

**Discussion by karttofer** [vidstack/player#1508] [Q&A]
Hello,

I’ve been implementing VidStack for YouTube Lives, which have mostly worked well in hiding the titles and logos of the streams. However, when I switch to another section, I get an error stating that the Provider has been destroyed. This makes me think that the application is trying to access the Provider after it has been destroyed (just a theory). The component is being rendered dynamically, FYI.
![Screenshot from 2024-11-16 10-24-09](https://github.com/user-attachments/assets/6576d0f2-80fd-4cd1-b57c-be8cd0e8cc91)

```js
import React from "react";
import '@vidstack/react/player/styles/default/theme.css';
import '@vidstack/react/player/styles/default/layouts/video.css';
import { useTranslation } from "react-i18next";

import { MediaPlayer, MediaProvider, Poster } from '@vidstack/react';
import { defaultLayoutIcons, DefaultVideoLayout } from '@vidstack/react/player/layouts/default';
import { Container } from "@chakra-ui/react";

const VideoPlayer = ({ height = '213px', videoConfigs }) => {
	const { t } = useTranslation();

	return (
		<Container w='100%' maxW="100%" h={height} p={0} className="video-player">
			<MediaPlayer {...videoConfigs}>
				<MediaProvider>
					<Poster
						className="vds-poster"
						src={thumbnails}
						alt={t("videoPlayer.alt")}
					/>
				</MediaProvider>
				<DefaultVideoLayout thumbnails={{ src: thumbnails }} icons={defaultLayoutIcons} />
			</MediaPlayer>
		</Container>
	);
};

export default VideoPlayer;|
```
```js
{
  videoConfigs: {
    viewType: "video",
    src: "https://www.youtube.com/watch?v=O9mOtdZ-nSk",
    streamType: "live",
  },
};

```

**Comment by rafael-jordao** [comment]
Same error here

**Reply by ChatterjeeArka** [reply]
@rafael-jordao  @karttofer Have you found any fix? I am facing the same issue!

**Reply by karttofer** [reply]
Hi @ChatterjeeArka - Not from my side, I think the owner of this repo isn't working on this thing anymore... Maybe one day someone can make a fork to fix it....

**Reply by ChatterjeeArka** [reply]
Okay @karttofer, I also thing the same. Dropping Vidstack for now, will check later if it's fixed!

**Comment by Apezdr** [comment]
For the poster, you didn't define your environment, need your package.json if React for sure.

**Reply by karttofer** [reply]
Hi ~ I'm not currently using this library anymore... But, I think for people like @ChatterjeeArka for example could be very helpful. When you mentioned the environment... What do you mean?

**Reply by Apezdr** [reply]
`need your package.json`

**Comment by sven-ra** [comment]
this is still happening whenever I navigate away from a page with Vidstack player.
- `next@16.1.6`
- `vidstack/react@1.12.13`
